src/util/Utils.js

```diff
@@ -0,0 +1,18 @@
+const Utils = {
+  convertStringToNumber(strings) {
+    const numbers = strings.map((string) => {
+      return Number(string);
+    });
```

Comment:
이 부분에 대해서 살펴볼 만한 점이 2가지가 있습니다!

1. 간소화하기
  - 간단한 함수는 이렇게 줄여볼 수도 있습니다.
  ```js
  // 간소화 1단계
     const numbers = strings.map(str => Number(str));
  ```

```suggestion
// 간소화 2단계
    const numbers = strings.map(Number);
```

2. [Number() vs parseInt](https://thisthat.dev/number-constructor-vs-parse-int/)

- Number("")와 parseInt("")의 차이점은 무엇일까요?